scripts/testnet/prerequisites.sh

```diff
@@ -56,7 +56,7 @@ fi
 cd $(dirname $ELRONDDIR)
 git clone git@github.com:ElrondNetwork/elrond-deploy-go.git
 
-if [ $PRIVATE_REPOS -eq 1 ]; then
+if [[ $PRIVATE_REPOS -eq 1 ]]; then
   git clone git@github.com:ElrondNetwork/elrond-proxy-go.git
```

Comment:
can you please take out this line outside of this if branch? elrond-proxy-go is no longer a private repo :)

Reply:
sure :) done